bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/validation/internal/ConfigDescriptionParameterValidator.java

```diff
@@ -12,20 +12,30 @@
 
 /**
  * The {@link ConfigDescriptionParameterValidator} can be implemented to provide a specific validation of a
- * {@link ConfigDescriptionParameter} and its actual value.
+ * {@link ConfigDescriptionParameter} and its value to be set.
  *
  * @author Thomas Höfer - Initial contribution
  */
 public interface ConfigDescriptionParameterValidator {
 
     /**
+     * Returns true, if the validator is to be executed for the given parameter and its given value, otherwise false.
+     *
+     * @param parameter the corresponding config description parameter
+     * @param value the value to be set for the config description parameter
+     *
+     * @return true, if the validator is to be executed for the given parameter and its given value, otherwise false
+     */
+    boolean isExecutable(ConfigDescriptionParameter parameter, Object value);
```

Comment:
Normally I would prefer to use "isApplicable".
But as long as the parameters of both functions are the same it doesn't matter.
I would still see a difference between "all fine" and "I doesn't validate anything, because I don't care" (in the current scenario there "result" is the same, but time will change... perhaps some time).